core/domain/opportunity_domain_test.py

```diff
@@ -172,34 +172,36 @@ def test_negative_content_count_fails_validation_check(self):
             self.valid_exp_opprtunity_summary.content_count = -5
             self._assert_validation_error(
                 self.valid_exp_opprtunity_summary,
-                'Expected content_count to be an non-negative integer, '
+                'Expected content_count to be a non-negative integer, '
                 'received -5')
 
     def test_same_language_for_need_and_assigend_voice_artist_fails_validation(
             self):
         need_voice_artist_languages = (
-            self.valid_exp_opprtunity_summary.need_voice_artist_in_languages)
+            self.valid_exp_opprtunity_summary
+            .need_voice_artist_in_language_codes)
         assigned_voice_artist_languages = (
-            self.valid_exp_opprtunity_summary.assigned_voice_artist_in_languages
-            )
+            self.valid_exp_opprtunity_summary
+            .assigned_voice_artist_in_language_codes)
 
         self.assertTrue(
             set(need_voice_artist_languages).isdisjoint(
                 assigned_voice_artist_languages))
         with self.mock_supported_audio_languages_context:
             self.valid_exp_opprtunity_summary.validate()
 
-            self.valid_exp_opprtunity_summary.need_voice_artist_in_languages = [
+            self.valid_exp_opprtunity_summary.need_voice_artist_in_language_codes = [ # pylint: disable=line-too-long
```

Comment:
"opportunity" is misspelled.

Use a temporary variable for self.valid_exp_opportunity_summary to avoid the pragma. In general, stop and think before you add a new pragma.